### DISCUSSION TITLE
Problems with GDL installer on windows

### DISCUSSION THREAD

**Discussion by AlexDevauchelle** [gnudatalanguage/gdl#941] [Q&A] [answered]
Hello! I'm having issues installing GDL on Windows 10. I'm following this guide : (https://github.com/gnudatalanguage/gdl/wiki/GDL-on-Windows).
My first issue is on the part "Build GDL" where the file "gdl/.ci/build_gdl_windows.msys" doesn't exist. So I guess the new file is "gdl/.ci/build_gdl.sh" but I need a confirmation for that.
Next, when I execute the commande "./gdl/.ci/build_gdl.sh build", it fails with fatals error : Invalid argument and no input files.
I've attached logs files and console logs so you can investigate them.
[CMakeError.log](https://github.com/gnudatalanguage/gdl/files/6472526/CMakeError.log)
[CMakeOutput.log](https://github.com/gnudatalanguage/gdl/files/6472527/CMakeOutput.log)
[CMakeCache.txt](https://github.com/gnudatalanguage/gdl/files/6472530/CMakeCache.txt)
[console.txt](https://github.com/gnudatalanguage/gdl/files/6472612/console.txt)

**Comment by pjb7687** [comment, marked as answer]
Hello,

If you just want to download GDL Windows binaries, then actually nowadays you don't have to build GDL by yourself. You can just download one of the latest automatic builds, e.g., https://github.com/gnudatalanguage/gdl/actions/runs/829874366

To build GDL, I had a look your console log, would like to suggest two things:

1. Would you try setting the directory paths UNIX style? e.g., export ROOT_DIR=/d/GDL
2. You seem to have MSYS version of GCC installed in your environment, and this is preferred in the default MSYS environment. You can go to the root of MSYS2 installation using file explorer, and run `mingw64.exe` to launch `MSYS2 MINGW64 Shell`. This will automatically set your environment to use MINGW binaries as default. Now you can try compiling GDL within this console window.

**Reply by AlexDevauchelle** [reply]
Thank you very much! This is exactly what I needed, so I used the automatic builds to install it.
